VIC UI download URL is wrong in install.bat

**VIC version:**

vic-v1.1.1_56a309fb.ova, downloaded from VMware.com

**Deployment details:**

Deployed VIC appliance, attempting to install the HTML5 vSphere Client plug-in on a VC 6.5 running in Windows Server 2012.

**Steps to reproduce:**

1. Update `vic/ui/vCenterForWindows/configs` file with VC IP address, URL to VIC appliance file server, and thumbprint.
2. Run `install.bat`.
3. Restart vSphere Client.

**Actual behavior:**

- `install.bat` reports success.
- `com.vmware.vic` appears as an extension in MOB for the VC.
- VIC UI doesn't appear in vSphere Client.
- `C:\ProgramData\VMware\vCenterServer\logs\vsphere-ui\logs\vsphere_client_virgo.log` includes `Error downloading https://10.153.199.213:9443/com.vmware.vic-1.1.1.zip. Make sure that the URL is reachable then logout/login to force another download.`

**Expected behavior:**

- That the plug-in should deploy to VC running on Windows. 
- Or, if not, there should be some error to state that installation didn't actually succeed without the user having to go searching through the logs.
 
**Logs:**

```[2017-05-19T07:38:12.947-07:00] [ERROR] vc-extensionmanager-pool-82  70000025 100003 200001 com.vmware.vise.vim.extension.VcExtensionManager                  Package com.vmware.vic was not installed! 
Error downloading https://10.153.199.213:9443/com.vmware.vic-1.1.1.zip. Make sure that the URL is reachable then logout/login to force another download. java.io.FileNotFoundException: https://10.153.199.213:9443/com.vmware.vic-1.1.1.zip
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1890)
	at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1885)
	at java.security.AccessController.doPrivileged(Native Method)
	at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1884)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1457)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1441)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:254)
	at com.vmware.vise.util.http.ConnectionManager.getResponseBodyAsStream(ConnectionManager.java:335)
	at com.vmware.vise.util.http.SimpleHttpClient.executeMethodResponseAsStream(SimpleHttpClient.java:146)
	at com.vmware.vise.vim.extension.VcExtensionManager.writePackageToFile(VcExtensionManager.java:945)
	at com.vmware.vise.vim.extension.VcExtensionManager.downloadPackage(VcExtensionManager.java:890)
	at com.vmware.vise.vim.extension.VcExtensionManager$2.call(VcExtensionManager.java:704)
	at com.vmware.vise.vim.extension.VcExtensionManager$2.call(VcExtensionManager.java:695)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at com.vmware.vise.util.concurrent.QueuingCachedThreadPool$QueueProcessor.run(QueuingCachedThreadPool.java:885)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.io.FileNotFoundException: https://10.153.199.213:9443/com.vmware.vic-1.1.1.zip
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1836)
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1441)
	at java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:480)
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:338)
	at com.vmware.vise.util.http.ConnectionManager.connect(ConnectionManager.java:278)
	at com.vmware.vise.util.http.SimpleHttpClient.connect(SimpleHttpClient.java:254)
	at com.vmware.vise.util.http.SimpleHttpClient.executeMethodResponseAsStream(SimpleHttpClient.java:145)
	... 11 common frames omitted
```

**Additional details as necessary:**

Investigation with @jooskim reveals an error in the `install.bat` script. There is a `v` missing in `SET PLUGIN_URL=%vic_ui_host_url%%key%-%version%.zip`. It should be `SET PLUGIN_URL=%vic_ui_host_url%%key%-v%version%.zip`.

--- a/ui/installer/vCenterForWindows/install.bat
+++ b/ui/installer/vCenterForWindows/install.bat
@@ -54,7 +54,7 @@ IF %target_vc_version% EQU 6.5 (
 )
 
 SET PLUGIN_FLAGS=--key %key:"=% --name %name% --version %version:"=% --summary %summary% --company %company%
-SET PLUGIN_URL=%vic_ui_host_url%%key%-%version%.zip
+SET PLUGIN_URL=%vic_ui_host_url%%key%-v%version%.zip
 IF /I %vic_ui_host_url% NEQ NOURL (
     IF %vic_ui_host_url:~-1,1% NEQ / (
         SET vic_ui_host_url=%vic_ui_host_url%/

